func-complete-check/package.json

```diff
@@ -4,31 +4,33 @@
   "private": false,
   "license": "GPL-3.0",
   "scripts": {
-    "build": "tsc",
+    "build": "yarn clean && tsc",
     "build:production": "yarn prestart",
     "watch": "tsc --w",
     "prestart": "yarn build && func extensions install",
     "start:host": "func start",
     "start": "yarn start:host & yarn watch",
-    "test": "yarn build && ./node_modules/.bin/jasmine",
-    "lint": "standard && tslint --project ."
+    "lint": "standard && tslint --project .",
+    "test": "yarn jest",
```

Comment:
Do we want to explicitly show how to get a code coverage report from jest?  E.g `yarn test --coverage`

Reply:
will add 🏆

Reply:
done